pom.xml

```diff
@@ -2777,7 +2777,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>8.14</version>
+            <version>8.18</version>
```

Comment:
Hi, @Fokko . Shall we use the latest `8.23` instead of `8.18`?

Reply:
Excellent idea @dongjoon-hyun, I'll push the commit right away